static/app/views/onboarding/onboardingController.tsx

```diff
@@ -1,24 +1,20 @@
-import {ComponentPropsWithoutRef, useEffect} from 'react';
+import {ComponentPropsWithoutRef} from 'react';
```

Comment:
Don't import this, use `React. ComponentPropsWithoutRef`

Reply:
why?

Reply:
bc you don't need to import it :)

`React` is in the global type namespace. I would like to add an eslint for this, but haven't looked into it